content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -55,9 +55,9 @@ By default a database will reject any operation that violates the referential in
 
 Referential actions are defined in the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attribute and map to the actions on the **foreign key constraint** in the underlying database. If you do not specify a referential action, [Prisma falls back to a default](#referential-action-defaults).
 
-The following model defines a 1-m relation between `User` and `Post` and a n-m relation between `Post` and `Tag`, with explicitly defined referential actions:
```

Comment:
Why drop the `with explicitly defined referential actions`? Not important to spell out? (Honestly don't know)

Reply:
Thanks, think this must have just got lost while rearranging, I put it back in